common/language.cpp

```diff
@@ -86,6 +86,7 @@ const LanguageDescription g_languages[] = {
 	{ "sr", "sr_RS", "Serbian", SR_SRB },
 	{ "sk", "sk_SK", "Slovak", SK_SVK },
 	{ "es", "es_ES", "Spanish", ES_ESP },
+	{ "eu", "eu_ES", "Basque", EU_EUS },
```

Comment:
The constant should be `EU_ESP` to stay consistent with the POSIX locale and to conform to ISO-3166 which does not contain EUS code. 
In addition, this list seems sorted by English language name, the line should be moved above.
Thanks!